saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/TextAnnotationFactory.scala

```diff
@@ -0,0 +1,55 @@
+/** This software is released under the University of Illinois/Research and Academic Use License. See
```

Comment:
You can reuse this in [SRL](https://github.com/IllinoisCogComp/saul/blob/10de99cc5833a87988f2dc25e03c2d9aacbeb3c7/saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/PopulateSRLDataModel.scala#L52-L64) too?